docs/source/gateway/index.rst

```diff
@@ -634,15 +636,20 @@ If a given provider does not provide support for a parameter, a clear message wi
 
 - **candidate_count** (Supported by OpenAI, Cohere): This parameter determines the number of alternative responses to generate. The range is from 1 to 5, and by default, it is set to 1.
 
-Alternate Query Parameters
---------------------------
-There are additional provider-specific parameters that will work (i.e., ``logit_bias`` (OpenAI, Cohere), ``frequency_penalty`` (OpenAI, Cohere), ``presence_penalty`` (OpenAI, Cohere), and ``top_k`` (Anthropic, Cohere)) with the exception of the following:
+Additional Query Parameters
+---------------------------
+In addition to the :ref:`standard_query_parameters`, you can pass any additional parameters supported by the route's provider as part of your query. For example:
 
-- **stream** is not supported. Setting this parameter on any provider will not work currently.
+- ``logit_bias`` (supported by OpenAI, Cohere)
+- ``top_k`` (supported by Anthropic, Cohere)
+- ``frequency_penalty`` (suppoorted by OpenAI, Cohere)
+- ``presence_penalty`` (supported by OpenAI, Cohere)
 
-- **top_k** is not supported if ``temperature`` is set. Use one or the other.
```

Comment:
``top_k`` is not explicitly rejected by the gateway if temperature is specified. it's up to the provider whether to allow this, so I think we should remove this from our docs.

Reply:
we do that with Anthropic on the gateway server side. I agree that it's not that important to put into the docs since it's confusing as a blanket statement.